src/rgw/rgw_rest_sts.h

```diff
@@ -14,11 +14,14 @@
 namespace rgw::auth::sts {
 
 class WebTokenEngine : public rgw::auth::Engine {
+  static constexpr string_view princTagsNamespace = "https://aws.amazon.com/tags";
```

Comment:
@mattbenjamin @adamemerson : The namespace under which tags have to be added in the IDP have been set to the one used by amazon currently. Should we change this to something specific to ceph?

Reply:
I don't think so. For things where we're offering the same functionality as Amazon we should use Amazon's namespace, that way things can operate on a Ceph or an AWS S3 without needing to be rewritten.